src/backend/tests/endpoints/test_search.py

```diff
@@ -109,3 +116,32 @@ def test_search_with_failing_save_event(service, monkeypatch):
     data = json.loads(response.data)
 
     check_response_record_matches_mock_crawler_search(data["record"], service.mock_record["john_doe"])
+
+
+def test_search_cache(service, monkeypatch): 
+    """
+    successful search results should cache
```

Comment:
This comment is unnecessary. Perhaps just rename the test `test_successful_search_cache`.